browser/src/control/Control.NotebookbarCalc.js

```diff
@@ -1092,9 +1094,9 @@ L.Control.NotebookbarCalc = L.Control.NotebookbarWriter.extend({
 				'vertical': 'true'
 			},
 			{
-				'type': 'bigtoolitem',
+				'type': 'bigcustomtoolitem',
 				'text': _UNO('.uno:HyperlinkDialog'),
-				'command': '.uno:HyperlinkDialog'
+				'command': 'hyperlinkdialog'
```

Comment:
With this change the button id has changed to something not ideal `Hyperlink...` can we add id here and set to `HyperlinkDialog` (as before)?

Reply:
HTML id

Reply:
fixed